components/patterns/navs/sidebar-mobile.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <aside
+    :class="{ open }"
+    class="nav-left fixed left-0 inset-y-0 z-10 w-full p-5 bg-white"
+  >
+    <div class="flex flex-col justify-start items-start mt-16 w-full">
+      <IconCloseSharp
+        @click.native="$emit('openNav')"
+        class="mr-2 mt-2 top-0 right-0 absolute w-8 h-8 text-mtl-black-400"
+      />
+      <mtl-h4 class="mb-4 text-mtl-green-400">
+        Navigation
+      </mtl-h4>
+      <mtl-text-info class="mb-4 pl-4">
+        <a
+          @click.prevent="$router.push(localePath('/'))"
+          :class="currentRoute == 'index' ? 'nuxt-link-active' : ''"
+          href=""
+        >
+          <span class="pb-1">{{ $t('home') }}</span>
+        </a>
+      </mtl-text-info>
+      <mtl-text-info class="mb-4 pl-4">
+        <a
+          @click.prevent="$router.push(localePath('/search'))"
+          :class="currentRoute == 'search' ? 'nuxt-link-active' : ''"
+          href=""
+        >
+          <span class="pb-1">{{ $t('events') }}</span>
+        </a>
+      </mtl-text-info>
+
+      <hr class="my-4" />
+      <mtl-h4 class="mb-4 text-mtl-green-400">
+        {{ $t('other') }}
+      </mtl-h4>
+      <mtl-text-info class="mb-4 pl-4">
+        <a
+          target="_blank"
+          rel="noopener noreferrer"
+          href="https://join.slack.com/t/vuemontreal/shared_invite/zt-6cmiy7iv-izbVijXeeDNcQOREPo8tWA"
+        >
+          {{ $t('joinslack') }}
```

Comment:
is that possible to have key like: `join_slack` or `join-slack`. It was hard to read `joinslack`.
The first time I was like ... `joins lack` ... what did he mean ^^

Reply:
sure